zuul.d/jobs.yaml

```diff
@@ -721,3 +721,18 @@
     cleanup-run: playbooks/new-tenant-juju/cleanup.yaml
     secrets:
       - serverstack_admin
+
+- job:
+    name: publish-charm
+    description: Publish a charm to charmhub
+    timeout: 1200
+    parent: tox
+    provides: charm
+    run: playbooks/charm/publish.yaml
+    secrets:
+      - serverstack_cloud
+      - charmhub_token
+    nodeset:
+      nodes:
+        - name: xenial-medium
+          label: xenial-medium
```

Comment:
Oh, we really need to move off xenial!  Could this not be at least focal?

Reply:
Changed it to focal-medium, thanks!